### PR TITLE
Fix linux build

### PR DESCRIPTION
**Pull request by blackrez**
I had some trouble to build on linux it is a proposition for fixing build on linux.
I'm not a expert on C++, I'm not sure of what I'm doing.
#7

### CODE REVIEW

File: geo/liblwgeom/lwgeom_geos_cluster.cpp [hunk 1]
```diff
@@ -37,8 +37,8 @@ static const int STRTREE_NODE_CAPACITY = 10;
 /* Utility struct used to accumulate items in GEOSSTRtree_query callback */
 struct QueryContext {
 	void **items_found;
-	uint32_t items_found_size;
 	uint32_t num_items_found;
+	uint32_t items_found_size;
```
Comment (handstuyennn) [inline geo/liblwgeom/lwgeom_geos_cluster.cpp:41]: Why does this need to change?

**Comment by handstuyennn** [comment]
Thanks for the PR! Looks good. I have a comment

**Comment by handstuyennn** [comment]
Thanks for the PR! Looks good. I have a comment

File: geo/liblwgeom/lwgeom_geos_cluster.cpp (hunk 1 above)
Comment (handstuyennn) [inline geo/liblwgeom/lwgeom_geos_cluster.cpp:41]: Why does this need to change?

Comment (handstuyennn) [inline geo/liblwgeom/lwgeom_geos_cluster.cpp:41]: Why does this need to change?

**Review by handstuyennn — COMMENTED** [review]
Thanks for the PR! Looks good. I have a comment

File: geo/liblwgeom/lwgeom_geos_cluster.cpp (hunk 1 above)
Comment (blackrez) [reply geo/liblwgeom/lwgeom_geos_cluster.cpp:41]: The compiler raise an exception about the order of arguments for this line :

https://github.com/handstuyennn/geo/blob/15ad5a691f55bbe263b4b5d1bf710cfe1fac09b3/geo/liblwgeom/lwgeom_geos_cluster.cpp#L187-L202

Comment (handstuyennn) [reply geo/liblwgeom/lwgeom_geos_cluster.cpp:41]: Thanks

**Pull request merged**